rhc-ose-ansible/inventory/inv

```diff
@@ -0,0 +1,71 @@
+# Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
```

Comment:
@JayKayy rather than adding an all encompassing inventory file, can you provide a minimal example of what's needed to apply the metrics role?